src/scenes/Authentication/Register/Register.tsx

```diff
@@ -26,10 +26,10 @@ export const Register = (props: Except<Props, 'onSubmit'>) => {
       const { data } = await register({
         variables: {
           input: {
-            displayFirstName: input.firstName,
-            displayLastName: input.lastName,
-            realFirstName: input.firstName,
-            realLastName: input.lastName,
+            displayFirstName: input.realFirstName,
+            displayLastName: input.realLastName,
+            realFirstName: input.displayFirstName,
+            realLastName: input.displayLastName,
```

Comment:
Hmm 👀

Reply:
I used a spread operator but the input had the confirmPassword so I got lazy

Reply:
Well I was attempting to point out that you've flipped the display & real fields.

But I think spreading would be better anyways.
```tsx
  const submit: Props['onSubmit'] = async ({ confirmPassword, ...input }) => {
    try {
      const { data } = await register({
        variables: {
          input: {
            ...input,
            timezone: DateTime.local().zone.name,
          },
        },
      });
```

Reply:
This is why you're the tech lead.  Fixed